### ISSUE TITLE
[Datadog] Eval/Setup USE Monitoring for EKS:Ingress

### ISSUE THREAD

**Issue by ph-One** [department-of-veterans-affairs/va.gov-team#49292]
# Description
As a platform engineer Datadog monitors and alerts need to be evaluated and/or setup for EKS:Ingress

# Acceptance Criteria
- [ ] EKS:Ingress (Traefik) is evaluated for monitoring needs
- [ ] Monitors are created, if appropriate
- [ ] Alerts are created, if appropriate

# Notes
- ELB that fronts Traefik/EKS
- The USE method for systems:
  - **U**tilization: The average time that the resource was busy serving work. We usually represent utilization as a percentage over time.
  - **S**aturation: The amount of work that a resource is unable to serve. We usually represent this metric as a queue length.
  - **E**rrors: The number of error events. We usually represent errors as a total count.

**Comment by ph-One** [comment]
Evaluate if this is already covered by EC2 monitors

**Comment by rsmithadhoc** [comment]
- U
    - [Platform - Traefik Open Connections](https://vagov.ddog-gov.com/monitors/113488)
- S
    - I did not find a metric that fit this category, there wasn't any queued requests as far as I could tell.
- E
    - [Platform: Traefik error rate](https://vagov.ddog-gov.com/monitors/113154)
        - Created as part of our [RED ticket](https://app.zenhub.com/workspaces/platform-tech-team-2-633efe4ca5a428e5294d7ade/issues/gh/department-of-veterans-affairs/va.gov-team/49293).

**Comment by it-harrison** [comment]
looks like the threshold on the Use monitor needs tweaking because it is alerting when the system is functioning normally.

**Comment by jhouse-solvd** [comment]
Note: This issue will carry into the next sprint. We've made significant progress, but there are several refinements needed to ensure comprehensive monitoring for EKS ([Clusters](https://app.zenhub.com/workspaces/platform-tech-team-2-633efe4ca5a428e5294d7ade/issues/gh/department-of-veterans-affairs/va.gov-team/50179) & [Ingress](https://app.zenhub.com/workspaces/platform-tech-team-2-633efe4ca5a428e5294d7ade/issues/gh/department-of-veterans-affairs/va.gov-team/47545)). 

Also, we consulted w/ Platform Tech Team 1 today to inform them of the monitoring that's been put in place for EKS. They are migrating Vets-API to EKS and will be adding monitors for parts of the application that are not yet covered.
